.github/workflows/ci.yml

```diff
@@ -68,21 +62,18 @@ jobs:
             python: '3.11'
             triplet: x64-osx-mixed
 
-          - runs-on: windows-latest
-            python: '3.7'
-            triplet: x64-windows
-          - runs-on: windows-latest
-            python: '3.8'
-            triplet: x64-windows
-          - runs-on: windows-latest
-            python: '3.9'
-            triplet: x64-windows
-          - runs-on: windows-latest
-            python: '3.10'
-            triplet: x64-windows
-          - runs-on: windows-latest
-            python: '3.11'
-            triplet: x64-windows
+#          - runs-on: windows-latest
```

Comment:
Any reason CI is disabled for Windows?

Reply:
See https://github.com/Farama-Foundation/Arcade-Learning-Environment/issues/515, we commented it out such that the CI passed in the meantime

Reply:
Fixed in #523